recipes/confindr/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "confindr" %}
+{% set version = "0.3.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "dae78a22b32cec2893553f624931b663c60d6362618b5db7d5d335d02a39e735" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  skip: True # [py27]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - biopython
+    - pysam
+    - pytest
+    - bbmap
+    - jellyfish
+    - blast
+    - mash >=2
+    - samtools
+    - pip
```

Comment:
can you remove this line ?

Reply:
Line has been removed in most recent commit

Reply:
thanks, looks good now, I will merge when the tests finish.

Reply:
Perfect, thanks!